redisinsight/values.yaml

```diff
@@ -0,0 +1,125 @@
+# Default values for redisinsight.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- The name of the environment
+platform: sandbox
+
+image:
+  # -- The repository of the image
+  repository: redis/redisinsight
+  # -- The pullPolicy used when pulling the image
+  pullPolicy: IfNotPresent
+  # -- The tag of the iamge. Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+server:
+  # -- The number of pods to be deployed for server
+  replicas: 1
+  # -- Specific values for the service resource
+  service:
+    # -- Service labels
+    labels: {}
+    # -- Service annotations
+    annotations: {}
+  # -- Specific values for the headless service resource
+  headless:
+    # -- Headless service labels
+    labels: {}
+    # -- Headless service annotations
+    annotations: {}
+  
+  # -- Collection of ports
+  ports:
+    # -- Port through which the UI is exposed
+    ui: 5540
+
+  # -- Extra environment variables to inject
+  extraEnvVars: {}
+
+# -- Ingress related values
+ingress:
+  # -- Whether to enable the ingress or not
+  enabled: false
+  # -- IngressClass used for the ingress
+  className: ""
+  # -- Ingress annotations
+  annotations: {}
+  hosts:
+    - host: redisinsight.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  # -- Tls settings
+  tls: []
+
+# -- Presistence related values
+persistence:
+  # -- Whether to enable persistence or not
+  enabled: true
+  # -- The name of the storageclass
+  className: ""
+  # -- Size of the Persistent Volume
+  size: 100Mi
+  # -- Annotations for the Persistent Volume Claim
+  annotations:
+    helm.sh/resource-policy: "keep"
```

Comment:
**suggestion:** I don't think we need this annotation, given we are deploying with ArgoCD and not Helm.